great_expectations/experimental/datasources/sources.py

```diff
@@ -154,7 +154,6 @@ def _register_datasource_and_factory_method(
 
     @classmethod
     def _register_assets(cls, ds_type: Type[Datasource], asset_type_lookup: TypeLookup):
-
```

Comment:
@alexsherstinsky this is the kind of change we should avoid making.
It needlessly adds this file to the list of changed files, and isn't obviously better than what existed previously. 
All these little changes add up and make it harder to navigate the git history of a file.

Reply:
@Kilo59 Sometimes it is worth incurring these changes early on in an effort that is still maturing so that we (or contributors) do not have to do a lot of them later.  Stylistic tidiness and logical organization along with verbose naming can reduce cognitive strain on those trying to learn the system.  Of course these are only my own aesthetic preferences and I can see how others may find more value in fewer changes; having said that, small changes can be understood and checked off quickly.   Not sure if these kind of formatting items need to be in a style guide, but happy to discuss.  Thank you!